### DISCUSSION TITLE
google 翻译配置的服务器url 怎么配置

### DISCUSSION THREAD

**Discussion by gufeng0** [YiiGuxing/TranslationPlugin#3898] [Q&A]
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/77660384/e52ff0b5-76fe-4bad-a37f-8bf2e6a3abc5)
请问这个怎么配

**Comment by YiiGuxing** [comment]
如果你有其镜像服务器，只需要输入镜像服务器站点URL就可以了，例如：[https://xxx.yyy.zzz]() （只是例子）

**Reply by gufeng0** [reply]
没有的话就用不了google 翻译吗

**Reply by YiiGuxing** [reply]
是的，可以使用梯子

**Reply by YiiGuxing** [reply]
> 请问下不勾选配置服务器会直接连接谷歌翻译原地址吗 开了梯子直接翻译 以及勾选服务器地址用谷歌翻译的原地址也翻译不了

勾选是表示使用配置的url，否则不使用。不能用应该是vpn配置不对。

**Reply by dest1yo** [reply]
> > 请问下不勾选配置服务器会直接连接谷歌翻译原地址吗 开了梯子直接翻译 以及勾选服务器地址用谷歌翻译的原地址也翻译不了
> 
> 勾选是表示使用配置的url，否则不使用。不能用应该是vpn配置不对。

我超好快的回 谢谢 但是我梯子开的是全局模式也不行 我这里浏览器里面连谷歌翻译使用都是没问题的

**Reply by YiiGuxing** [reply]
> > > 请问下不勾选配置服务器会直接连接谷歌翻译原地址吗 开了梯子直接翻译 以及勾选服务器地址用谷歌翻译的原地址也翻译不了
> > 
> > 勾选是表示使用配置的url，否则不使用。不能用应该是vpn配置不对。
> 
> 我超好快的回 谢谢 但是我梯子开的是全局模式也不行 我这里浏览器里面连谷歌翻译使用都是没问题的

可以看看 #2051、#2296。

**Reply by dest1yo** [reply]
> > > > 请问下不勾选配置服务器会直接连接谷歌翻译原地址吗 开了梯子直接翻译 以及勾选服务器地址用谷歌翻译的原地址也翻译不了
> > > 
> > > 
> > > 勾选是表示使用配置的url，否则不使用。不能用应该是vpn配置不对。
> > 
> > 
> > 我超好快的回 谢谢 但是我梯子开的是全局模式也不行 我这里浏览器里面连谷歌翻译使用都是没问题的
> 
> 可以看看 #2051、#2296。

非常感谢！ 才明白原来vpn的配置指的是规则有问题 很感谢耐心回复

**Comment by yang-chun** [comment]
我的梯子是谷歌的插件 这个怎么设置?
